ext_emconf.php

```diff
@@ -11,7 +11,7 @@
     'author_email' => 'lolli@schwarzbu.ch',
     'constraints' => [
         'depends' => [
-            'typo3' => '12.99.99-13.99.99',
+            'typo3' => '12.4.99-13.99.99',
```

Comment:
but it should be 12.4.0- ... shouldn't it?

Reply:
^ I'd say yes - 12.4.0-13.99.99 should the way to go ;)